controller/contact_controller.go

```diff
@@ -25,3 +26,65 @@ func PostContantsHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+func GetContantsHandler(w http.ResponseWriter, r *http.Request) {
+
+	data, err := service.GetContact(r.Context())
+
+	if err != nil {
+		log.Println("Internal error", err.Error())
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.Write(data)
+}
+
+func PutContantsHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
```

Comment:
This is what we send back to the client so it should be at the end of the function. Move this line to the last line of the function.